src/helpers/dataframe/sort.ts

```diff
@@ -29,7 +29,10 @@ export function sortableDataFrame<M extends Obj, C extends Obj>(
     sortable: sortableColumns.has(name),
     metadata: structuredClone(metadata), // Create a deep copy of the column metadata to avoid mutating the original
   }))
-  const metadata = structuredClone(data.metadata) // Create a deep copy of the metadata to avoid mutating the original
+  let metadata: any = structuredClone(data.metadata) // Create a deep copy of the metadata to avoid mutating the original
+  if (options && 'exclusiveSort' in options) {
+    metadata = { ...metadata ?? {}, exclusiveSort: options.exclusiveSort }
+  }
```

Comment:
in the methods `fetch`, `getCell` and `getRowNumber`, we might want to add a validation on `orderBy`, and throw if an array with more than 1 element is passed?

Reply:
it can be another helper validation function, or done by extending the existing validation function for `orderBy`

Reply:
also, we need to validate the HighTable orderBy prop (and add tests for it): what occurs if a user passes orderBy with two entries: should we throw? should we keep the first entry and ignore the other ones?